malaria24/ona/models.py

```diff
@@ -149,16 +153,33 @@ def get_html_email_content(self):
             'ona/html_email.html', self.get_email_context())
 
 EHP = 'EHP'
+CASE_INVESTIGATOR = 'CASE_INVESTIGATOR'
 MANAGER_DISTRICT = 'MANAGER_DISTRICT'
 MANAGER_PROVINCIAL = 'MANAGER_PROVINCIAL'
 MANAGER_NATIONAL = 'MANAGER_NATIONAL'
 
+PROVINCES = [
```

Comment:
Because we already have historical data in the facilities model where provinces are text fields and these need to match.